src/MoBi.Presentation/Tasks/ObservedDataTask.cs

```diff
@@ -384,6 +385,26 @@ private void addDimensionsTo(ColumnInfo columnInfo)
          }
       }
 
+      private void addPredefinedMoleculesForImporter(MetaDataCategory metaDataCategory)
+      {
+         if (metaDataCategory == null)
+            return;
+
+         metaDataCategory.ShouldListOfValuesBeIncluded = true;
+
+         foreach (var molecule in allMolecules())
+         {
+            var molWeightParameter = molecule.AllParameters().FirstOrDefault(x => x.Name == AppConstants.Parameters.MOLECULAR_WEIGHT);
```

Comment:
you could use 
`var molWeightParameter = molecule.Parameter(AppConstants.Parameters.MOLECULAR_WEIGHT)`

Reply:
done